### PR TITLE
Update plugins.json

### PR DESCRIPTION
**Pull request by creecros**
## creecros/Group_assign
- added js was double submiting forms, this removes the second submit
- creecros/Group_assign#37 Separates `assignee:` filter and adds `allassignees:` filter
  - using `allassignees:Username` or `allassignees:Name` will find all tasks assigned to that user regardless of how they have been assigneed, whether in the group or in Other Assignees or Assignee.

## creecros/Customizer
- 1.11.2 revisions were causing an infinite loop on installs using external logins, i.e. oauth, LDAP
  - Reverting to prevent infinite loop
- creecros/Customizer#88

## eSkiSo/Subtaskdate
- fix: illegal offset
  - `PHP message: PHP Warning:  Illegal string offset 'id' in /var/www/app/plugins/Subtaskdate/Model/SubtaskCalendarModel.php on line 80`

## kenlog/UpdateNotifier
- fix: remove empty line and create table 'if not exists'
  - kenlog/UpdateNotifier#11 

## creecros/MarkdownPlus
- first release
- Adds EmojiOne `:shortname:` and Unicode conversions to Markdown in Kanboard. 
- Switch between oldschool github style emoji's or newschool emojione emojis, the choice is yours.
- :shortname: autocomplete
- supports check boxes
- inline html
- markdown extra

### CODE REVIEW

File: plugins.html [hunk 1]
```diff
@@ -360,6 +360,11 @@ <h1>Kanboard Plugins</h1>
                     <p>Use the Mailgun API to send emails and create tasks from emails.</p>
                     <p><em>By Frédéric Guillot</em></p>
                 </dd>
+                <dt><a href="https://github.com/creecros/MarkdownPlus">Encrypted Content</a></dt>
```
Comment (fguillot) [inline plugins.html]: The title seems wrong. Bad copy and paste?

File: plugins.json [hunk 2]
```diff
@@ -119,15 +119,27 @@
...
         "compatible_version": ">=1.0.42"
     },
+    "MarkdownPlus": {
+        "title": "MarkdownPlus",
+        "version": "1.0.0",
+        "author": "Craig Crosby",
+        "license": "MIT",
+        "description": "Improved Markdown, with check boxes, emoji shortcode, inline html, etc...",
+        "homepage": "https://github.com/creecros/MarkdownPlus",
+        "readme": "https://raw.githubusercontent.com/creecros/MarkdownPlus/master/README.md,
```
Comment (fguillot) [inline plugins.json]: There is missing double quote here (invalid JSON).

**Review by fguillot — CHANGES_REQUESTED** [review]

File: plugins.html (hunk 1 above)
Comment (creecros) [reply plugins.html]: Ya, it started out as just emojis then I got bored and added more stuff, changed the name, yadda yadda. I'll change it in a minute

Comment (creecros) [reply plugins.html]: all set

File: plugins.json (hunk 2 above)
Comment (creecros) [reply plugins.json]: and all set

**Comment by NannerHammer** [comment]
Just an FYI, this caused a login loop on my installation while using standard local auth, not just external. Reverting solved the issue for me. Odd thing is, I didn't get an infinite loop. It would loop somewhere between once and 20 times. Pulled a few conversations from my NIDS sensor logs to confirm (between the webserver and a host attempting to auth). I can provide those if you'd like proof of the issue. 

The login conversations consisted of somewhere between 2 and 20 login attempts that redirected to /login, followed by successful auth.

Failed login security was disabled while I tested this, as was practically every other option that isn't essential. Uninstalled every plugin but Customizer.

**Comment by creecros** [comment]
1.11.2, I hope you mean. This is a PR for 1.11.3, which if you are having issues with, wouldn't be because of Customizer, since all login redirects were removed. 

If you mean 1.11.2, I believe you, don't need proof. I tried to fix a minor issue with a simple redirect, and didn't have the the foresight to see that it might cause different setups to behave differently than my own. The loop was intentionally coded, and never meant to loop more than once, but apparently it took 20 loops for your system to accurately get the user I'd. In all, it was a bad solution to a problem, that I don't think anyone other than myself has noticed.

**Pull request merged**